File: 02 Digits and Numbers - Foundation Programs/program-06.cpp
// Write a program that takes a 2-digit number as input and displays it in reverse order (e.g., 45 becomes
// 54).

#include<iostream>
using namespace std;
int main()
{
    int number,rev_digit;
    cout<<"Enter 2-digit number : ";
    cin>>number;
    
    rev_digit=(number%10)*10;
    rev_digit=rev_digit+(number/10);

    cout<<"reverse order : "<<rev_digit;

    return 0;
}

File: archit_vishwakarma/Level-1/03-Digits-And-Numbers/program-06.py
n = input("Enter a number : ")

try:
    n = int(n)
    if n < 100 and n > 9:
        
        a = n%10
        b = a*10
        reverse = b + (n//10)

        print(f"reverse of two digit : {reverse}")
        
    else:
        print("Enter a Two Digit Number")
except ValueError:
    print("enter a valid number")